features/support/hooks.rb

```diff
@@ -27,3 +27,7 @@
   block.call
   Rack::Attack.enabled = true
 end
+
+Before("@stub_pda_contracts_endpoint") do
```

Comment:
I have added this hook as a way to resolve a number of failing integration tests - not sure if this is the best approach?